auslib/test/admin/views/test_releases.py

```diff
@@ -438,7 +438,7 @@ def testReleasePostRejectedURL(self):
     def testDeleteRelease(self):
         ret = self._delete("/releases/d", qs=dict(data_version=1))
         self.assertStatusCode(ret, 200)
-        ret = dbo.releases.t.count().where(dbo.releases.name == 'd').execute().first()[0]
+        ret = dbo.releases.count(where=[dbo.releases.name == 'd'])
```

Comment:
I take it this change is to fix warnings such as:
`  /app/.tox/py27/lib/python2.7/site-packages/sqlalchemy/sql/expression.py:1783: SAWarning: The IN-predicate on "rules.rule_id" was invoked with an empty sequence. This results in a contradiction, which nonetheless can be expensive to evaluate.  Consider alternative strategies for improved performance.`

And I'm guessing that using plain `t.count()` is source of them?

Reply:
This one is not deprecated anymore: https://docs.sqlalchemy.org/en/latest/changelog/migration_12.html?highlight=predicate#the-in-not-in-operator-s-empty-collection-behavior-is-now-configurable-default-expression-simplified

The count fix is about https://docs.sqlalchemy.org/en/rel_1_1/core/selectable.html?highlight=fromclause%20count#sqlalchemy.sql.expression.FromClause.count